tunnistamo/settings.py

```diff
@@ -29,6 +29,11 @@
     MEDIA_URL=(str, '/media/'),
     NODE_MODULES_ROOT=(str, os.path.join(BASE_DIR, 'node_modules')),
 
+    COOKIE_SECURE=(bool, True),
+    COOKIE_PREFIX=(str, 'tunnistamo'),
+    # For scoping cookies to path
+    PATH_PREFIX=(str, '/'),
```

Comment:
Now that I read this again, I think this name might be too ambiguous. I mean, if you see that in the Azure library, you wouldn't know which path that affects. So maybe `COOKIE_PATH_PREFIX`?

Reply:
Or just `COOKIE_PATH` as the property's name in the cookie is `Path`?

Reply:
I picked that up from HELERM and decided to use that for some consistency. But is indeed too generic. Will change.

Reply:
I guess Ville named it prefix as it's used in two settings and they in turn might set different value after the prefix. Which is improbable but possible. But in that case there should be two environment values for those too. :)

Reply:
Just copied this from helerm without thinking. Technically it is prefix, as per http specification (or MDN at least). But it is not called a prefix

Reply:
I'm not invested in the name, just wanted to have them under a single parameter as I've never seen them setup to separate values.

Reply:
`CSRF_AND_SESSION_COOKIE_PATH` then? 😄

Reply:
That is rather long and thus typo prone. How about:
DJANGO_COOKIE_NAME_PREFIX : name prefix for all Django cookies
DJANGO_COOKIE_PATH: path for all Django cookies
DJANGO_COOKIE_SECURE: secure flag for all Django cookies
right now those cookies are {prefix}-sessionid and {prefix}-csrf

I'll also volunteer to push them into our documentation on dev.hel.fi, which also is in need of some care.

Reply:
Perhaps the `DJANGO_` prefix isn't needed as it's not used in any other name either.

Reply:
Fair enough. I'll shorten them.

Reply:
Otherwise I liked the names just as proposed: COOKIE_NAME_PREFIX, COOKIE_PATH, COOKIE_SECURE. Those are my favorites :smile:

Reply:
Names are changed as per conversation.